core/shared/src/main/scala/org/http4s/multipart/Multiparts.scala

```diff
@@ -36,22 +36,17 @@ trait Multiparts[F[_]] {
 
 object Multiparts {
 
-  /** Creates a `scala.util.Random` and provides Multiparts around it.
+  /** Creates a `cats.effect.std.Random` and provides Multiparts around it.
     * This instance can be shared, or is cheap enough to create closer
     * to where Multipart values are generated.
     */
   def forSync[F[_]](implicit F: Sync[F]): F[Multiparts[F]] =
-    F.delay(new Random()).map(fromScalaRandom[F])
+    Random.scalaUtilRandom[F].map(fromRandom[F])
```

Comment:
FTR I feel weird about this methods. `Multiparts` doesn't need `Sync`. It so happens that you can get a `Random` with a `Sync`, but to me that seems to be no business of `Multiparts`.

Reply:
Yeah, It's just a convenience method.  Since it already existed, I'd be inclined to keep it.  The important thing is we can create Multiparts without that constraint now.